pepys_import/file/importer.py

```diff
@@ -22,13 +22,20 @@ def __init__(self, name, validation_level, short_name, datafile_type, default_pr
         self.errors = None
         self.error_type = None
 
-        self.do_recording = True
+        # By default all importers will record extractions to a highlighted
+        # HTML file, but not record them to the database
+        self.highlighting_level = "html"
```

Comment:
@robintw - maybe I'm a crusty old Java programmer who is far from `pythonic`, but...

I would always declare these strings as constants, and refer to the constant in the code (to avoid [magic strings](https://deviq.com/antipatterns/magic-strings).  IMHO that avoids typos, and easily lets us find (for example) all of the places where a particular highlighting mode is used.

Reply:
I thought you might say that! In python code people tend to use strings rather than constants for these sorts of things, but I can switch to using constants if you'd prefer - I'm happy either way.

Reply:
Do you have a position on this as we move forward @mew-nsc ? Ours will be the new team-culture from the end of the week ;-)

Reply:
"Named constant?" or "Would this be better as an enum?" are some of my most common code review comments... It looks like an enum might make sense here, but I'm conscious that the Python convention is to use strings - I'm guessing we should probably stick with the language conventions?

Reply:
We do use named constants elsewhere (eg. for validation levels - which is a similar setting that is configured for an importer) - so we could use them here.
I'm wondering whether switching to use constants for this PR might be a good task for @mew-nsc, as it would give him a good excuse to look through all the code in a reasonable amount of detail and see how it all fits together. I'm happy to do it myself, but just wondered if that might be useful.

Reply:
Agree that it would be useful to give me a better understanding of the extractions. Will take a look.

Reply:
Pushed the change (went for an enum based approach). Please shout if you have any concerns